docs/source/daml/resource-management/acs.rst

```diff
@@ -200,9 +200,9 @@ When you need to read a contract, or act on a single contract in multiple ways,
        create this with quantity = quantity + sum quantities
  
 
-Use separate templates for large payloads that change rarely and require minimum access, and for fields that change with almost every action, to optimize resource consumption for multiple business transactions. 
+Separate templates for large payloads that change rarely and require minimum access from those for fields that change with almost every action. This optimizes resource consumption for multiple business transactions. 
 
-This batching approach makes updates in one transaction submission rather than having separate transactions for each update. Note: this option can increase latency a little bit and may increase the possibility of command failure but this can be avoided. For example:
+This batching approach makes updates in one transaction submission rather than requiring separate transactions for each update. Note: this option can increase latency a little bit and may increase the possibility of command failure but this can be avoided. For example:
```

Comment:
```suggestion
This batching approach makes updates in one transaction submission rather than requiring separate transactions for each update. Note: this option can cause a small increase in latency and may increase the possibility of command failure but this can be avoided. For example:
```